src/lib/headers/Interval.js

```diff
@@ -52,7 +52,8 @@ class Interval extends React.PureComponent {
     return {
       ...this.props.getIntervalProps({
         interval: this.props.interval,
-        ...props
+        ...props,
+        style: Object.assign({}, this.getIntervalStyle(), props.style)
```

Comment:
could you please pass the `props.style` to `getIntervalStyle` and spread them there?